marketplace-service/src/test/java/com/axonivy/market/controller/ProductDetailsControllerTest.java

```diff
@@ -149,4 +175,60 @@ private ProductDetailModel createProductMockWithDetails() {
     mockProductDetail.setContactUs(false);
     return mockProductDetail;
   }
+
+  private ProductJsonContent mockProductJsonContent() throws JsonProcessingException {
```

Comment:
schema repeated twice in 2 different test classes. Consider to create a mock file for it